src/components/App/App.module.css

```diff
@@ -1,8 +1,24 @@
+body {
+	background-color: #FCE4EC;
+}
+
 .wrap {
-	margin: auto;
-	width: 650px;
+	display: flex;
+	justify-content: space-between;
+    margin: auto;
+    width: 800px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)